index.html

```diff
@@ -1,39 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <!-- metadata -->
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-    <!-- title and favicon -->
-    <title>Web App</title>
-    <link rel="icon" type="image/x-icon" href="assets/alchemy-favicon.png" />
-
-    <!-- js -->
-    <script type="module" src="app.js"></script>
-
-    <!-- fonts -->
-    <!-- see https://fonts.google.com/ -->
-
-    <!-- css -->
-    <link rel="stylesheet" href="styles/reset.css" />
-    <link rel="stylesheet" href="styles/global.css" />
-    <!-- add link for page specific css -->
-
-</head>
-
-<body>
-    <header>
-        <img class="logo" src="assets/html_css_js.png" alt="logo">
-        <h1>Web App</h1>
-    </header>
-
-    <main>
-
-    </main>
-
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <!-- metadata -->
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+        <!-- title and favicon -->
+        <title>Halloween Hunter</title>
+        <link rel="icon" type="image/x-icon" href="assets/cauldron.png" />
+
+        <!-- js -->
+        <script type="module" src="app.js"></script>
+
+        <!-- fonts -->
+        <!-- see https://fonts.google.com/ -->
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Bungee+Spice&family=Dangrek&display=swap');
+        </style>
+
+        <!-- css -->
+        <link rel="stylesheet" href="styles/reset.css" />
+        <link rel="stylesheet" href="styles/global.css" />
+        <!-- add link for page specific css -->
+    </head>
+
+    <body>
+        <header>
+            <img class="logo" src="assets/cauldron.png" alt="logo" />
+            <h1>Halloween Hunter</h1>
+        </header>
+
+        <main>
+            <section class="banner" id="score">You have not hunted any monsters.</section>
+            <section class="banner" id="outcome">Let's get Hunting!</section>
+
+            <section class="witch" id="witch">
+                <p id="witch hp" class="stat">20 HP</p>
+                <img id="witch image" class="girl" src="assets/witch.png" />
+            </section>
```

Comment:
so right here you're hard-coding the HTML to display the player instead of using state and render to populate it so I can't give you points